config/handles_ir_communities_collections.yml

```diff
@@ -0,0 +1,1145 @@
+handle_ir_communities_collections
```

Comment:
Replace line (1) with `handles_ir_communities_collections`. Then, update https://github.com/osulp/Scholars-Archive/blob/master/app/controllers/scholars_archive/handles_controller.rb#L51 and https://github.com/osulp/Scholars-Archive/blob/master/app/controllers/scholars_archive/handles_controller.rb#L62